site/archive/archiveTypes.ts

```diff
@@ -0,0 +1,12 @@
+export interface UrlAndMaybeDate {
+    url: string
+    date?: Date
+}
+export interface ArchiveSiteNavigationProps {
+    archiveDate: Date
+    liveUrl?: string
+    previousVersion?: UrlAndMaybeDate
+    nextVersion?: UrlAndMaybeDate
```

Comment:
I've implemented `nextVersion` already in design - but it's not wired up, i.e. never shown.